lib/pbench/server/api/__init__.py

```diff
@@ -20,10 +22,20 @@
 from pbench.server import PbenchServerConfig
 from pbench.common.exceptions import BadConfig
 from pbench.server.utils import filesize_bytes
+from pbench.server.api.resources.pbench_users import RegisterUser, LoginAPI, LogoutAPI, GetUser
 
 ALLOWED_EXTENSIONS = {"xz"}
 
 app = None
+bcrypt = None
+db = None
+
+""""
+A storage engine to save revoked tokens. if speed is the primary concern, redis is a good option. but if data 
+persistence is more important, postgres can be another good option too. Here I am using an in memory store to get 
+something working.
```

Comment:
Rather than store "revoked" tokens, we should store generated/live tokens, with expiration dates. A lookup after the expiration date would automagically delete the token. TBD whether it would fail the authorization immediately or wait until the next access.

Reply:
See the comment above about token expiration